DigitalLearningSolutions.Web/ViewModels/Register/SummaryViewModel.cs

```diff
@@ -5,20 +5,11 @@
 
     public class SummaryViewModel: IValidatableObject
     {
-        public SummaryViewModel(string firstName, string lastName, string email, string centre, string jobGroup)

```

Comment:
Hmm.. Is there a reason we've removed the constructor and made all the fields optional?

Reply:
Models need to have a parameterless constructor, it's a limitation of MVC. When I tried to get around this (by providing a parameterless constructor as well as one with all the properties) there were still issues I would just have to overcomplicate things to fix so I decided it would be simpler to have a default constructor and use the object initialiser.

Reply:
Hmm.. I'm pretty sure we have the parameterless + parameter constructors elsewhere (e.g. ForgotPasswordViewModel). What were the issues when you tried both?

Reply:
Ah, the errors I get are that it says all the other fields in the viewmodel are required even though the only we care about is if the terms and conditions box is ticked. Seeing that ForgotPasswordViewModel works makes me think it's trying to construct the viewmodel from what's being posted by the form. I could add hidden inputs for each of the other fields, though that still feels a bit hacky.